src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -50,232 +50,231 @@ gaugesTable = {}
-  orchid                    = { 218, 112, 214 },
-  pale_goldenrod            = { 238, 232, 170 },
-  PaleGoldenrod             = { 238, 232, 170 },
-  pale_green                = { 152, 251, 152 },
-  PaleGreen                 = { 152, 251, 152 },
-  pale_turquoise            = { 175, 238, 238 },
-  PaleTurquoise             = { 175, 238, 238 },
-  pale_violet_red           = { 219, 112, 147 },
-  PaleVioletRed             = { 219, 112, 147 },
-  papaya_whip               = { 255, 239, 213 },
-  PapayaWhip                = { 255, 239, 213 },
-  peach_puff                = { 255, 218, 185 },
-  PeachPuff                 = { 255, 218, 185 },
-  peru                      = { 205, 133, 63 },
-  pink                      = { 255, 192, 203 },
-  plum                      = { 221, 160, 221 },
-  powder_blue               = { 176, 224, 230 },
-  PowderBlue                = { 176, 224, 230 },
-  purple                    = { 160, 32, 240 },
-  royal_blue                = { 65, 105, 225 },
-  RoyalBlue                 = { 65, 105, 225 },
-  red                       = { 255, 0, 0 },
-  rosy_brown                = { 188, 143, 143 },
-  RosyBrown                 = { 188, 143, 143 },
-  saddle_brown              = { 139, 69, 19 },
-  SaddleBrown               = { 139, 69, 19 },
-  salmon                    = { 250, 128, 114 },
-  sandy_brown               = { 244, 164, 96 },
-  SandyBrown                = { 244, 164, 96 },
-  sea_green                 = { 46, 139, 87 },
-  SeaGreen                  = { 46, 139, 87 },
-  seashell                  = { 255, 245, 238 },
-  sienna                    = { 160, 82, 45 },
-  sky_blue                  = { 135, 206, 235 },
-  SkyBlue                   = { 135, 206, 235 },
-  slate_blue                = { 106, 90, 205 },
-  SlateBlue                 = { 106, 90, 205 },
-  slate_gray                = { 112, 128, 144 },
-  SlateGray                 = { 112, 128, 144 },
-  slate_grey                = { 112, 128, 144 },
-  SlateGrey                 = { 112, 128, 144 },
-  snow                      = { 255, 250, 250 },
-  steel_blue                = { 70, 130, 180 },
-  SteelBlue                 = { 70, 130, 180 },
-  spring_green              = { 0, 255, 127 },
-  SpringGreen               = { 0, 255, 127 },
-  tan                       = { 210, 180, 140 },
-  thistle                   = { 216, 191, 216 },
-  tomato                    = { 255, 99, 71 },
-  turquoise                 = { 64, 224, 208 },
-  violet_red                = { 208, 32, 144 },
-  VioletRed                 = { 208, 32, 144 },
-  violet                    = { 238, 130, 238 },
-  wheat                     = { 245, 222, 179 },
-  white                     = { 255, 255, 255 },
-  white_smoke               = { 245, 245, 245 },
-  WhiteSmoke                = { 245, 245, 245 },
-  yellow                    = { 255, 255, 0 },
-  yellow_green              = { 154, 205, 50 },
-  YellowGreen               = { 154, 205, 50 }
-}
+color_table = color_table or {}
```

Comment:
You could have just done this:
```lua
color_table = color_table or {
-- all the colors that were there already
}
```
I think this style is less readable.

Reply:
Would what you are proposing work if the user (or the C++ core) have added their own colours to the table? I intended for this bit to only reset the colours that are listed/defined in that part of the script and NOT clobber any other entries under different keys...

Reply:
But this code doesn't get run many times right, it's only code in C++ that does the updating. This code is run before any user code is ever run and then never again - unless `resetProfile()` is called, where it is still run before user code :grin:

Reply:
I'd still like to have this chunk reverted since the change makes it harder for me to see if there were actual content changes.